### PR TITLE
added malyalam text

### PR DESCRIPTION
**Pull request by arpit7714**
### Issue Reference
Added Malayalam text

### Summarize
translate the 
english to malayalam text in 
privaterescuecamp_form.html

please review @TribeAsteraea

### CODE REVIEW

**Comment by arpit7714** [comment]
solved #843

File: mainapp/templates/mainapp/privaterescuecamp_form.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 {% block content %}
 <h1 class="text-center">Enter New Camp Details</h1>
-<h2 class="text-center">--</h2>
+<h2 class="text-center">പുതിയ ക്യാമ്പിൽ വിശദാംശങ്ങൾ  നൽകുക </h2>
```
Comment (biswaz) [inline mainapp/templates/mainapp/privaterescuecamp_form.html]: It should "campinte" right?

Comment (riginoommen) [reply mainapp/templates/mainapp/privaterescuecamp_form.html]: @biswaz yes

Comment (arpit7714) [reply mainapp/templates/mainapp/privaterescuecamp_form.html]: @biswaz   it should be "campinte" instead of Camp or  else.
i am unable to get you

Comment (biswaz) [reply mainapp/templates/mainapp/privaterescuecamp_form.html]: its ക്യാമ്പിന്റെ , not ക്യാമ്പിൽ

**Pull request merged**